src/components/slideshow/slideshow.js

```diff
@@ -137,7 +168,7 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
                 if (!actionButtonsOnTop) {
                     html += '<div class="slideshowBottomBar hide">';
 
-                    html += getIcon('pause', 'btnSlideshowPause slideshowButton', true, true);
+                    html += getIcon('play_arrow', 'btnSlideshowPause slideshowButton', true, true);
```

Comment:
why change icon?

Reply:
The slideshow was auto-playing before, but is now is paused by default, except on Chromecast (But I'm not sure the OSD shows up on Chromecast, since it's non-interactive).

Reply:
If `play` will go after this
https://github.com/jellyfin/jellyfin-web/blob/18d1305ead762f01202ab6e6d59c590ef95ecce4/src/components/slideshow/slideshow.js#L285-L286
then I think this change is valid in all cases